youngseo/app.js

```diff
@@ -53,6 +53,116 @@ app.post("/users/signup", async (req, res) => {
     res.status(201).json({ message: "userCreated" });
 });
 
+app.post("/posts", async (req, res) => {
+    const { title, content, content_image, user_id } = req.body
+
+    await appDataSource.query(
+        `INSERT INTO posts(
+            title,
+            content,
+            content_image,
+            user_id
+        ) VALUES (?, ?, ?, ?);
+        `,
+        [ title, content, content_image, user_id ]
+    ); 
+    res.status(201).json({ message: "postCreated" });
+});
+
+app.get("/posts", async(req, res) => {
+    await appDataSource.query(
+        `SELECT 
+            users.id AS userId,
+            users.profile_image AS userProfileImage,
+            posts.id AS postingId,
+            posts.content_image AS postigImageUrl,
+            posts.content AS postingContent
+        FROM users 
+        INNER JOIN posts ON users.id = posts.user_id`
+            ,(err, rows) => {
+        res.status(200).json({data:rows});
+            });
+});
+
+
+app.get("/posts/:userId", async (req, res) => {
+    const userid = req.params.userId
+
+    const [user] = await appDataSource.query(
+        `SELECT 
+            users.id AS userId,
+            users.profile_image AS userProfileImage
+        FROM users
+        WHERE users.id = ${userid}`);
+    
+    const posts = await appDataSource.query(
+        `SELECT 
+            posts.id AS postingId,
+            posts.content_image AS postingImageUrl,
+            posts.content AS postigContent
+        FROM posts
+        INNER JOIN users ON users.id = posts.user_id
+        WHERE users.id = ${userid}`);
+
+    user['postings'] = posts;
+
+    res.status(200).json({ data: user })
```

Comment:
```suggestion
    return res.status(200).json({ data: user })
```
response를 보내주고 더이상 처리할 코드가 없다면 return 을 작성해주세요!